plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/util.ts

```diff
@@ -71,32 +71,30 @@ export const parseRepoUrl = (
     );
   }
 
-  if (type === 'bitbucket') {
-    if (host === 'bitbucket.org') {
-      if (!workspace) {
-        throw new InputError(
-          `Invalid repo URL passed to publisher: ${repoUrl}, missing workspace`,
-        );
+  const repo: string = parsed.searchParams.get('repo')!;
+  switch (type) {
```

Comment:
It's nice to see some cleanup of this utility function. Would you be able to add a couple of unit tests in `utils.test.tsx` for the `parseRepoUrl` function. Huge karma boost for tests ❤️  ⭐